src/color-picker/color-picker.css

```diff
@@ -4,6 +4,13 @@
 	gap: 1em;
 }
 
+#space_picker {
+	flex: 0 1 100%; /* occupy the entire row */
```

Comment:
Why?

Reply:
I didn't want to change the flex layout we already use to something else or add a wrapper around the sliders and color swatch to position the space picker above the rest of the elements of the picker, so I did this. Now, I switched to the grid layout.